src/Api/AdminConsole/Controllers/PoliciesController.cs

```diff
@@ -85,6 +85,23 @@ public async Task<PolicyDetailResponseModel> Get(Guid orgId, int type)
         return new PolicyDetailResponseModel(policy);
     }
 
+    [HttpGet("{type}/policy-status")]
+    public async Task<bool> GetPolicyStatusAsync(Guid orgId, int type)
```

Comment:
Why not use `PolicyType` type directly? 
```suggestion
    public async Task<bool> GetPolicyStatusAsync(Guid orgId, PolicyType type)
```